app/views/campaigns/my_interviews/_my_team_interviews_interviews_list.html.erb

```diff
@@ -112,4 +112,124 @@
     </div>
   </div>
 
+  <!--  mobile-->
```

Comment:
@harold93 putting this in a partial? The risk is that a dev in the future would edit the desktop and not scroll until the very bottom of the view, therefore missing out on the very existence of a mobile version of the view.